scripts/github-actions/create-pull-request.js

```diff
@@ -17,6 +17,84 @@ const createPullRequest = async ({ context, github, baseBranch, branchName, desc
       reviewers,
     })
   }
+  // Get node information about the PR since adding it to the Firewatch Project Board Beta requires use of graphQL
```

Comment:
Why doesn't the existing workflow that adds PRs/Issues to the board work for these PRs? Is it because we are ignoring collaborator PRs? I think it makes more sense to update that workflow to work for these PRs.

Reply:
There are certainly ways to make this better but I was going with the easiest route to fix the problem.  I don't really have the time to refactor all of them at the moment.

Reply:
Is seems like it would be way easier to update the [triage_add_to_project](https://github.com/cypress-io/cypress/blob/develop/.github/workflows/triage_add_to_project.yml) workflow. Am I missing something?

Reply:
I believe we would need to update the PR creation scripts to output a variable and pass it to the triage_add_to_project and have that accept that input.